cells/lib/ops/mkStandardOCI.nix

```diff
@@ -68,19 +68,43 @@ in
       then debugOperable
       else operable;
 
-    setupLinks = cell.ops.mkSetup "links" [] ''
-      mkdir -p $out/bin
-      ${runtimeEntryLink}
-      ${debugEntryLink}
-      ${livenessLink}
-      ${readinessLink}
-    '';
+    setupLinks =
+      cell.ops.mkSetup "links" [
+        {
+          regex = "/bin";
+          mode = "0555";
+        }
+      ] ''
+        mkdir -p $out/bin
+        ${runtimeEntryLink}
+        ${debugEntryLink}
+        ${livenessLink}
+        ${readinessLink}
+      '';
+
+    setupBareMinimumEnv =
+      cell.ops.mkSetup "env" [
+        {
+          regex = "/tmp";
+          mode = "0777";
+        }
+      ] ''
+        mkdir -p $out/tmp
+        # Setup certificates
+        cp -r ${nixpkgs.cacert}/* $out
```

Comment:
`mkStandardOCI` serving P95, do we have indication or suspicion that this would break within the 95th percentile of usage instances/scenarios? (since we're in lack of data, this is armchair guesswork)

Reply:
There is a `dockerTools.caCertificates`, but I'm not sure if there is any difference.

Reply:
yeah, I tried `dockerTools.caCertificates`. Result seemed identical

Reply:
@gytis-ivaskevicius Can you maybe confirm if [this nixpkgs example](https://github.com/NixOS/nixpkgs/blob/d925734d3bb7f12924e6016cd33222684b5435f5/pkgs/build-support/docker/examples.nix#L727) suffers from the same issue on local docker?

Reply:
@blaggacao 
```
bash-5.2# ls -la /
total 0
drwxrwxr-x   1 0 0   88 May 16 09:15 .
drwxrwxr-x   1 0 0   88 May 16 09:15 ..
-rwxr-xr-x   1 0 0    0 May 16 09:15 .dockerenv
dr-xr-xr-x   1 0 0 1040 Jan  1  1980 bin
drwxr-xr-x   5 0 0  360 May 16 09:15 dev
dr-xr-xr-x   1 0 0   68 May 16 09:15 etc
dr-xr-xr-x   1 0 0    8 Jan  1  1980 lib
dr-xr-xr-x   1 0 0   18 Jan  1  1980 libexec
dr-xr-xr-x   1 0 0   10 Jan  1  1980 nix
dr-xr-xr-x 600 0 0    0 May 16 09:15 proc
dr-xr-xr-x   1 0 0    6 Jan  1  1980 share
dr-xr-xr-x  13 0 0    0 May 15 15:28 sys
bash-5.2# ls -la /etc
total 16
dr-xr-xr-x 1 0 0  68 May 16 09:15 .
drwxrwxr-x 1 0 0  88 May 16 09:15 ..
-rw-r--r-- 1 0 0  13 May 16 09:15 hostname
-rw-r--r-- 1 0 0 174 May 16 09:15 hosts
lrwxrwxrwx 1 0 0  12 May 16 09:15 mtab -> /proc/mounts
dr-xr-xr-x 1 0 0   6 Jan  1  1980 pki
-rw-r--r-- 1 0 0 974 May 16 09:15 resolv.conf
dr-xr-xr-x 1 0 0  10 Jan  1  1980 ssl
bash-5.2# ls -la /etc/ssl/certs
total 8
dr-xr-xr-x 1 0 0 64 Jan  1  1980 .
dr-xr-xr-x 1 0 0 10 Jan  1  1980 ..
lrwxrwxrwx 1 0 0 87 Jan  1  1980 ca-bundle.crt -> /nix/store/7vwvn3d5yb5yb3vdv9iv4321ckx4paf9-nss-cacert-3.86/etc/ssl/certs/ca-bundle.crt
lrwxrwxrwx 1 0 0 87 Jan  1  1980 ca-certificates.crt -> /nix/store/7vwvn3d5yb5yb3vdv9iv4321ckx4paf9-nss-cacert-3.86/etc/ssl/certs/ca-bundle.crt
```

Reply:
Thanks a lot!

At the same time, this is strange. `coreutils` and `caCertificates` only account for a portion of the observable file system hierarchy.

Where would the rest come from?

Reply:
This left me still wondering if `caCertificates` is autonomously solving the top level linking problem or if some external effect of the particular environment is at play.

The bug is only really fixed if `caCertificates` all on its own is capable of dodging the issue with top level linking of `/etc` and docker's, then destructive, overlay files in `/etc`.

Reply:
> Where would the rest come from?
coreutils contains `bin`, `share` and `libexec`
bash contains `bin` and `lib`
`dockerTools.caCertificates` contains `etc`